tools/build/build.coffee

```diff
@@ -0,0 +1,47 @@
+fs = require('fs')
+path = require('path')
+
+_ = require('lodash')
+consolidate = require('consolidate')
+
+Metalsmith = require('metalsmith')
+dynamic = require('metalsmith-dynamic')
+inplace = require('metalsmith-in-place')
+HbHelper = require('@resin.io/doxx-handlebars-helper')
+{ defaultPartialsSearch } = require('@resin.io/doxx-utils')
+
+plugins = require('./ms-plugins')
+utils = require('../lib/utils')
+
+rootDir = path.resolve(__dirname, '..')
+
+# monkey-patch
+originalCompile = HbHelper.Handlebars.compile
+HbHelper.Handlebars.compile = (program, options = {}) ->
+  options.noEscape = true
+  return originalCompile(program, options)
+
+HbHelper.registerConsolidate(consolidate, importName: 'include')
+require('./hb-helpers')(HbHelper.Handlebars)
+
+metalsmith = Metalsmith(rootDir)
+.source('src')
+.destination(utils.destDir)
+.use(defaultPartialsSearch())
+.use(dynamic({
+  dictionaries: path.join(rootDir, 'dicts')
+  populateFields: [ '$partials_search' ]
```

Comment:
I know you've copied it from my example, but I'm not sure this line is actually needed. Can you check please?